clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -16,21 +16,21 @@
  */
 package org.apache.kafka.clients;
 
-import java.util.HashSet;
-import java.util.Set;
-
-import java.util.stream.Collectors;
 import org.apache.kafka.common.errors.AuthenticationException;
 import org.apache.kafka.common.utils.ExponentialBackoff;
 import org.apache.kafka.common.utils.LogContext;
+
 import org.slf4j.Logger;
```

Comment:
According to description, the import order is 

1. org.apach.kafka.*
1. javax.*
1. java.*
1. *

Is ```org.slf4j.Logger``` in wrong order?

Reply:
@chia7712 Oh, the description I had at first on 2nd April is outdated; [After the discussion](https://lists.apache.org/thread.html/rf6f49c845a3d48efe8a91916c8fbaddb76da17742eef06798fc5b24d%40%3Cdev.kafka.apache.org%3E), we concluded that [the following three-group ordering](https://issues.apache.org/jira/browse/KAFKA-10787) would be better:

- `kafka`, `org.apache.kafka`
- `com`, `net`, `org`
- `java`, `javax`